data-plane/benchmarks/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/filter/NotFilterBenchmark.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.knative.eventing.kafka.broker.dispatcher.impl.filter;
+
+import dev.knative.eventing.kafka.broker.dispatcher.Filter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.subscriptionsapi.ExactFilter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.subscriptionsapi.NotFilter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.subscriptionsapi.PrefixFilter;
+import dev.knative.eventing.kafka.broker.dispatcher.impl.filter.subscriptionsapi.SuffixFilter;
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.core.v1.CloudEventV1;
+import java.net.URI;
+import java.util.Map;
+
+public class NotFilterBenchmark {
+    public static CloudEvent event() {
+        return CloudEventBuilder.v1()
+                .withId("abcdefghijklmnop")
+                .withSource(URI.create("http://localhost"))
+                .withType("com.github.pull.create")
+                .withDataSchema(URI.create("/api/schema"))
+                .withDataContentType("testContentType")
+                .withSubject("testSubject")
+                .build();
+    }
```

Comment:
This part of the code is duplicated across all these 3 files. If these classes are in a shared package or module, consider abstracting this to a utility class or a common parent class maybe?